content/posts/irods-development-update-june-2022.md

```diff
@@ -0,0 +1,297 @@
+Title: iRODS Development Update: June 2022
+Date: 2022-06-28 12:00
+Author: Kory Draughn
+Slug: irods-development-update-june-2022
+Status: published
+
+
+First off, for people who may have missed it, iRODS 4.3.0 is available now! You can read all about it by visiting this [blog post](https://irods.org/2022/06/irods-4-3-0-is-released/).
```

Comment:
just hyperlink "iRODS 4.3.0 is available now", and lose the last sentence... this isn't a blog(!) :P